File: backend/CadastroLivros/Data/Entities/Preco.cs
namespace CadastroLivros.Data.Entities
{
    public class Preco
    {
        public long CodP { get; set; }
        public long CodL { get; set; }
        public long CodFC { get; set; }
        public string FormaCompra { get; set; }
        public decimal Valor { get; set; }
    }
}

File: frontend/cadastro-livro-ui/src/app/models/preco.ts
export class Preco {
    constructor(codP: number = 0, codFC: number = 0, formaCompra: string = '', valor: number = 0 ) {
        this.codP = codP;
        this.codFC = codFC;
        this.formaCompra = formaCompra;
        this.valor = valor;
    }

    public codP: number;
    public codFC: number;
    public formaCompra: string;
    public valor: number;
}